dev/prod_builders.json

```diff
@@ -845,6 +845,48 @@
          "repo": "flutter",
          "task_name": "win_web_tool_tests",
          "flaky": true
+      },
```

Comment:
Can you remove `dev/devicelab/manifest.yaml` as well?

Reply:
done.